docs/developerguide.rst

```diff
@@ -155,7 +155,7 @@ After running ``minikube start`` on the remote host.
      type: "kubernetes"
      config: "./development-kubeconfig.yaml"
      config_context: "minikube"
-     version: "v1.11.2"
+     version: "v1.12.0"
```

Comment:
The CERN deployment has `v1.12.1` I believe, see the previously cited link. Can you rather use v1.12.1 everywhere?